packages/exo/package.json

```diff
@@ -33,6 +33,7 @@
   },
   "dependencies": {
     "@endo/env-options": "^0.1.4",
+    "@endo/eventual-send": "^0.17.6",
```

Comment:
This seems like incorrect layering.  Can you please put `getMethodNames` somewhere better, like `patterns` or `pass-style`?  If you can't remove it from `@endo/eventual-send`, at least deprecate it there, but make a new version in the better place.